crates/symbolicator/src/services/symbolication.rs

```diff
@@ -824,22 +826,26 @@ impl SymCacheLookup {
         });
     }
 
+    /// Returns the original `CompleteObjectInfo` list in its original sorting order.
+    fn into_inner(mut self) -> Vec<CompleteObjectInfo> {
+        self.inner.sort_by_key(|entry| entry.module_index);
+        self.inner
+            .into_iter()
+            .map(|entry| entry.object_info)
+            .collect()
+    }
+
     #[tracing::instrument(skip_all)]
     async fn fetch_symcaches(
-        self,
+        &mut self,
         symcache_actor: SymCacheActor,
-        request: SymbolicateStacktraces,
-    ) -> Self {
-        let mut referenced_objects = BTreeSet::new();
-        let SymbolicateStacktraces {
-            stacktraces,
-            sources,
-            scope,
-            ..
-        } = request;
-
+        scope: Scope,
+        sources: Arc<[SourceConfig]>,
+        stacktraces: &[RawStacktrace],
+    ) {
+        let mut referenced_objects = HashSet::new();
```

Comment:
same here, i think this is storing an index?  i assume the same logic applies but i'm curious what it is